Filters/exceptions.txt

```diff
@@ -2,6 +2,8 @@
 ! Once you enable private DNS, Android 9 starts resolving random domains looking like
 ! `*-dnsotls-ds.metric.gstatic.com` (for instance, `a5a6380f-dnsotls-ds.metric.gstatic.com`).
 !
+! https://forum.adguard.com/index.php?threads/fresh-install-on-moto-x4-one-specific-app-is-not-working-with-adguard.36898
+@@||api.prod.zedge.net^|
```

Comment:
which rule and filter block it? Cannot find.

Reply:
![Screenshot_20200113-200201](https://user-images.githubusercontent.com/34266271/72283696-99d60100-363f-11ea-9777-f0128c90c057.png)

Reply:
Or better exclude that wide rule?

Reply:
we have exclusions for similar cases(two groups of IP). Please add like existing. Cannot show at the moment:(

Reply:
![image](https://user-images.githubusercontent.com/8361299/72285888-b4fb3d80-364c-11ea-9061-d6a11e9adbbe.png)

Reply:
Done.